common-files/.github/workflows/jenkins-security-scan.yml

```diff
@@ -0,0 +1,23 @@
+# More information about the Jenkins security scan can be found at the developer docs: https://www.jenkins.io/doc/developer/security/scan/
```

Comment:
Please reference `https://www.jenkins.io/redirect/jenkins-security-scan/` from outside jenkins.io for URL stability.